collision_detection/src/collision_robot.cpp

```diff
@@ -254,3 +254,14 @@ void collision_detection::CollisionRobot::getScale(std::vector<moveit_msgs::Link
 void collision_detection::CollisionRobot::updatedPaddingOrScaling(const std::vector<std::string> &links)
 {
 }
+
+collision_detection::CollisionResult::DistanceDetailedMap collision_detection::CollisionRobot::distanceSelfDetailed(const robot_state::RobotState &state) const
```

Comment:
Might want to drop in a comment, that the default implementation is being implemented for backwards compatibility (assuming that a default implementation does not make sense).